src/form.html

```
@@ -88,6 +107,16 @@ <h2>BackVault Setup</h2>
             <input type="password" name="master_password" required />
             <label>Backup File Password</label>
             <input type="password" name="file_password" required />
+            <div class="org-section">
+                <label>Organization Export Mode</label>
+                <select name="org_export_mode">
+                    <option value="single" selected>Single combined file</option>
+                    <option value="multiple">Separate files per organization</option>
+                </select>
+                <label>Organization IDs (optional)</label>
+                <input type="text" name="organization_ids" placeholder="Comma-separated org IDs (leave empty for all)" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)